tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  future: {
+    // removeDeprecatedGapUtilities: true,
+    // purgeLayersByDefault: true,
+  },
+  purge: [],
+  theme: {
+    extend: {},
+  },
+  variants: {},
+  plugins: [],
+}
```

Comment:
Kindly R&D how to integrate tailwind with create react app, with purge options etc.